### DISCUSSION TITLE
Cannot run Blocks.sln with VS2022

### DISCUSSION THREAD

**Discussion by Chen-Haoyan** [microsoft/AirSim#4671] [Support Q&A]
AirSim Version/#commit: v1.8.1 - Windows
UE/Unity version: 5.0.3
OS Version: win10

3 error：
error C2672: “common_utils::Utils::isDefinitelyLessThan”: 未找到匹配的重载函数 （No matching overloaded function found）
error C2027: 使用了未定义类型“physx::PxVehicleWheels” （Type not defined）
error C2027: 使用了未定义类型“physx::PxVehicleDrive” （Type not defined）

**Comment by Leon-xc** [comment]
The exact same thing happened to me.

**Comment by brz-hub** [comment]
Have you solved this problem?

**Reply by Leon-xc** [reply]
AirSim cannot work with UE5， try UE4.27

**Comment by xxEoD2242** [comment]
Hi all,

At [Colosseum](https://github.com/CodexLabsLLC/Colosseum), we have a `ue5` branch that will allow Blocks to be run with UE5. The setup and integration is different and I am writing updated documentation for that now.

![UE5_AirSim](https://user-images.githubusercontent.com/32143517/201423834-e0d52318-b5e3-43af-bf0e-ed23e0659be0.png)

**Reply by xxEoD2242** [reply]
Basically, you need to go to VS Studio 2022 and install the .Net framework 3.0.1. Then, regenerate the project files for Blocks and then use the .sln file to build the entire project from scratch.

So far everything works as expected!

**Reply by Leon-xc** [reply]
> Hi all,
> 
> At [Colosseum](https://github.com/CodexLabsLLC/Colosseum), we have a `ue5` branch that will allow Blocks to be run with UE5. The setup and integration is different and I am writing updated documentation for that now.
> 
> ![UE5_AirSim](https://user-images.githubusercontent.com/32143517/201423834-e0d52318-b5e3-43af-bf0e-ed23e0659be0.png)

thank you for the great job. looking forward for your documents.

**Reply by Leon-xc** [reply]
Hi ~
I cloned from Colosseum and try to rebuild it following the offical Airsim docs. But the block.uprojects still failed to open by UE5. Why is that ? Do i need to do something else?
By the way, i used VS2022 and .net framework 4.7.2 ( i cannot  find .net 3.0.1).
Thanks for answering.
